ui/Screen/Onboarding.svelte

```diff
@@ -38,9 +40,11 @@
 
   const onCreateIdentity = async (handle, passphrase) => {
     try {
+      screen.lock();
+      requesting = true;
+      notification.info("Creating the identity...");
```

Comment:
Should we maybe get rid of this? For the release build this is rather quick and the notification may persist after the identity has been created.

Reply:
The notification is c key store is created before the key store creation request, which takes a long time.

Reply:
Jinx!